src/utils.jl

```diff
@@ -423,3 +436,38 @@ tosecond(x::Hour) = Float64(Dates.value(Second(x)))
 tosecond(x::Minute) = Float64(Dates.value(Second(x)))
 tosecond(x::T) where {T<:DatePeriod} = Float64(Dates.value(Second(x)))
 tosecond(x::T) where {T<:TimePeriod} = x / convert(T, Second(1))
+
+"""
+adjacent_nodes_at_link(graph)
```

Comment:
indent for code formatting, also directly below

Reply:
applied indent for code formatting.